frontend/app/(footer)/dashboard/page.tsx

```diff
@@ -95,21 +86,19 @@ export default function Dashboard() {
             })
             .finally(async () => {
               await sleep(500);
+              setLoading(false);
             })
             .catch(() => {
               router.push("/error");
             });
         }
       }
       if (location.trim() !== "") {
-        setSrcLocation(location);
         if (triggerFilterApply === undefined) {
-          setTriggerFilterApply(false);
+          setTriggerFilterApply(true);
```

Comment:
from what i can see, this field makes like no sense, the places its used in checks if its undefined or not so true false makes no difference or am i trippin

Reply:
You right